hedera-node/hedera-app/src/main/java/com/hedera/node/app/workflows/handle/validation/AttributeValidatorImpl.java

```diff
@@ -91,7 +92,8 @@ public void validateExpiry(long expiry) {
                 context.configuration().getConfigData(EntitiesConfig.class).maxLifetime();
         final var now = context.consensusNow().getEpochSecond();
         final var expiryGivenMaxLifetime = now + maxEntityLifetime;
-        validateTrue(expiry > now && expiry < expiryGivenMaxLifetime, INVALID_EXPIRATION_TIME);
+        final var impliedExpiry = expiry == NA ? expiryGivenMaxLifetime : expiry;
```

Comment:
We reverted the changes that have been made [there](https://github.com/hashgraph/hedera-services/commit/da0acc45019fe41c3098d369aa96dcd1eea69a4e#diff-b4e0fb3193e48f7f7a3a0cca1b407cb5dcfd4fd1d46b8132f8521f6d195c3c46), because the logic when the `expiry` is not explicitly set was failing in the tests. 

@Neeharika-Sompalli , Can you please share if these changes were removed intentionally?

CC: @tinker-michaelj  @petreze

Reply:
@fchitakova  Same here. Could you please point us to the tests that were failing with previous change. Thanks

Reply:
The preivous failing tests were `createTokenWithDefaultExpiryAndEmptyKeys` and `createTokenWithFixedFeeThenTransferAndAssessFee`

Reply:
Makes sense!